### ISSUE TITLE
require multiple restarts

### ISSUE THREAD

**Issue by comfysage** [comfysage/chaivim#1]
errors multiple times before working.

errors in order:
- core module not found at core.setup
- attempt to index local module (nil) when booting keymaps
- telescope not found

**Comment by jojo7682345** [comment]
Could not get it to work, even after several restarts.

```
Error detected while processing /home/dms/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/dms/.config/nvim/init.lua:2: module 'core' n
ot found:
        no field package.preload['core']
        no file './core.lua'
        no file '/usr/share/luajit-2.1.0-beta3/core.lua'
        no file '/usr/local/share/lua/5.1/core.lua'
        no file '/usr/local/share/lua/5.1/core/init.lua'
        no file '/usr/share/lua/5.1/core.lua'
        no file '/usr/share/lua/5.1/core/init.lua'
        no file './core.so'
        no file '/usr/local/lib/lua/5.1/core.so'
        no file '/usr/lib/lua/5.1/core.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
        [C]: in function 'require'
        /home/dms/.config/nvim/init.lua:2: in main chunk
```

**Comment by comfysage** [comment]
@jojo7682345 what does your init.lua look like? ( or whatever file you use to install and configure sentinel )

**Comment by comfysage** [comment]
should be fixed in 730ebd60.
the issue was with cloning the preloaded modules AFTER adding the path to `vim.opt.runtimepath`.
ill update the docs later

**Comment by comfysage** [comment]
the docs (and README) have been updated in 6b278b2b
